Make runner compatible with wide range of SDK revisions

mhofman opened the pull request:
When I started regenerating the corpus of stats, I realized that the runner and loadgen did not run correctly against both older and newer SDK revisions (I was working against a somewhat outdated version).

In particular:
- Newer SDKs do not provide sufficient RUN/BLD to pay for Zoe fees and let the loadgen complete multiple cycles
- When killing the chain while under load (middle of a mailbox delivery), the validator node would stall on restart (https://github.com/Agoric/agoric-sdk/issues/4115)
- The stats of shorter stages would be impacted by increasing setup time that was performed during stage 1, causing these stats to be unreliable

The first issue is fixed by provisioning and starting the local-solo manually, which required hooking into the cli to figure out the right path to use across revisions.

The second issue is addressed by winding down the loadgen before shutting everything down. The issue has since been fixed in the agoric-sdk (https://github.com/Agoric/agoric-sdk/pull/4116), but the workaround is left for older revisions, and is made configurable if we want to test for regressions.

For the last issue, the default config moves the wallet and loadgen deployments to stage 0, which is now truly a full setup stage.

As always, best reviewed commit-by-commit.

Stacks on top of the stats work (#36).

michaelfig reviewed (approved):
LGTM!

Straightforward, but lots of details.  I tried mainly to make sure your way of starting up the chain and provisioning matched with my understanding, which it does.

mhofman commented on runner/lib/tasks/local-chain.js:
```diff
@@ -38,31 +47,42 @@ const chainGoArgvMatcher = getArgvMatcher([/(?:sh|node)$/, /ag-chain-cosmos$/]);
+      await fs.stat(name);
+      return true;
+    } catch (e) {
+      return false;
```
Agreed, but to be fair this is ripped straight from `agoric-cli`:

https://github.com/Agoric/agoric-sdk/blob/98d414ad35f82ecea4d962caed7f4986ce0157d3/packages/agoric-cli/src/start.js#L122-L129

mhofman commented:
@michaelfig I addressed your feedback in https://github.com/Agoric/testnet-load-generator/pull/37/commits/ec6244195b03ad5287de9d421d57f2a113a05ca8 and made a small further change in https://github.com/Agoric/testnet-load-generator/pull/37/commits/1e66f9ec839038234d73b09093ed81a213feaad6.

Will carry your previous approval, squash and merge.
